src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -42,7 +42,7 @@ import scala.util.{ Failure, Success, Try }
  * @param id        the ID of the deposit
  */
 case class DepositDir private(draftBase: File, user: String, id: UUID) extends DebugEnhancedLogging {
-  val bagDir: File = draftBase / user / id.toString / "bag"
+  val bagDir: File = draftBase / user / id.toString / badDirName
```

Comment:
```suggestion
  val bagDir: File = draftBase / user / id.toString / bagDirName
```
Also in all other usages and declarations, lol